### DISCUSSION TITLE
Welcome to Dear ImGui Discussions!

### DISCUSSION THREAD

**Discussion by ocornut** [ocornut/imgui#3718] [Old/Archived discussions]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by epezent** [comment]
Hi everyone, I'm Evan! I [research](http://evanpezent.com/) haptic devices for virtual and augmented reality. Over the years, I've experimented with a number of GUI frameworks and visualization approaches to help tune and debug hardware in realtime, ranging from quick Python GUIs/plots over interprocess communications to full-blown native Qt apps. Without question, ImGui has been the **simplest and most enjoyable** solution of the bunch, because it enables me to 1) write short lived GUI code right in the middle of control algorithms with only a few lines of code, and 2) scale said code to create more sophisticated apps when needed. 

As a part of this community, I've authored a 2D plotting extension to ImGui called [ImPlot](https://github.com/epezent/implot) which has been steadily growing over the past few months. ImGui and ImPlot together have been a gamechanger in the way I and my colleagues rapidly iterate on robotic hardware and software. I cannot recommend it enough!
